anchore_engine/analyzers/manager.py

```diff
@@ -30,7 +30,7 @@ def apply_hints(analyzer_report, unpackdir):
     # hints processing.
     for engine_entry in utils.content_hints(unpackdir=unpackdir):
         pkg_type = engine_entry.get("type")
-        if pkg_type:
+        if pkg_type and pkg_type in syft.modules_by_engine_type:
```

Comment:
I think an info message here would be good to indicate to the user that there are hints entry types not processed by the syft handler.

Reply:
@zhill that might be a bit confusing. The syft handler is not applying hints for certain package types, but they _are_ getting applied by the analyzers themselves

Reply:
@dakaneye I could move it to debug only?